Styled login screen

Minmi345 opened the pull request.

Minmi345 committed:
Change credential location

Change in the storing of credentials file

Minmi345 committed:
Merge branch 'main' of https://github.com/b3h3m0th/pineapple-planner

Minmi345 committed:
Merge branch 'main' of https://github.com/b3h3m0th/pineapple-planner

Minmi345 committed:
Merge branch 'main' of https://github.com/b3h3m0th/pineapple-planner

Minmi345 committed:
Design of login page

Minmi345 committed:
Implemented register UI + slightly ui for settings

Minmi345 committed:
xxxx

b3h3m0th commented on PineapplePlanner/PineapplePlanner.Infrastructure/FirestoreService.cs:
```diff
@@ -16,7 +16,9 @@ public FirestoreDb FirestoreDb
-            GoogleCredential credential = GoogleCredential.FromFile("firebase-adminsdk-key.json");
+            // Credential file should be stored in %APPDATA%/PineapplePlanner/firebase-key.json
+            String credentialPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "PineapplePlanner\\firebase-key.json");
+            GoogleCredential credential = GoogleCredential.FromFile(credentialPath);
```
Please stick to the original approach. If you want to change the project configuration, please explain why and adapt the documentation in the README.md.

b3h3m0th commented on PineapplePlanner/PineapplePlanner.Infrastructure/FirestoreService.cs:
```diff
@@ -16,7 +16,9 @@ public FirestoreDb FirestoreDb
         {
-            GoogleCredential credential = GoogleCredential.FromFile("firebase-adminsdk-key.json");
+            // Credential file should be stored in %APPDATA%/PineapplePlanner/firebase-key.json
+            String credentialPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "PineapplePlanner\\firebase-key.json");
```
It is generally recommended to use `string` instead of `String` as a data type.

b3h3m0th commented on PineapplePlanner/PineapplePlanner.UI/Pages/Login.razor.css:
```diff
@@ -0,0 +1,51 @@
+
+
+.login-container {
+    background-color: antiquewhite;
```
Try to use hex codes and avoid using colors by names as they are different from browser to browser. Hex codes are definite values.

Minmi345 committed:
Design for navigational bar

Done task 48 (add placecholder for the picture)
Designed navigation bar
Changed named colours to hex colours
deleted "log in" link in nav.razor, because it's on every page anyways

Minmi345 committed:
Colour names -> hex code

minor changes in colours

b3h3m0th committed:
reverted firestore service config

Minmi345 committed:
Merge branch 'UI-design' of https://github.com/b3h3m0th/pineapple-planner into UI-design
